src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
Hmm, that's really interesting. I looked at the [same file on GitHub] and it gets rendered fine. Perhaps it's a bug with `mdbook` or `highlight.js` where the highlighting isn't applied properly?

[file on GitHub]: https://github.com/gurgalex/rust-ffi-guide/blob/386aaed65d0c9ea9f3612d113eb71b428b9ee395/src/bindgen/index.md